### ISSUE TITLE
Paste widget is confusing and problematic for screen reader users

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#239369]
From @jooyoungseo: 
1. Paste an image in a markdown document
2. Try to access the code actions, `ctrlCmd+.` like the one from the vision extension, to refine alt text.
3. Instead, the paste menu receives focus.
4. Hit `escape`.  Hit `ctrlCmd+.` again to try to access code actions, 🐛  the paste menu is activated.

cc @isidorn 

My suggestion:

Make the paste menu a code action with a submenu for the options, prioritizing it at the top.

**Comment by mjbvz** [comment]
I don't think we should combine these as they are different UX flows. Instead I think we need to clearly communicate to screen reader users when they in various combinations of the following states:

- Paste widget visible

- Code actions available 

- Snippet mode. In your example, pressing esc in step 4 only exits snippet mode for the paste edit. To dismiss the paste widget here you'd actually need to press `esc` twice: once  to exit the snippet and once more to dismiss the paste widget. This is the expected behavior 

Let me know how we should indicate this to screen reader users

**Comment by meganrogge** [comment]
If we cannot merge this with the code actions (still the route i'd suggest), we will need to add a different keybinding for the paste menu and also an accessibility signal (cc @afre100) to indicate that it's visible. 

I am concerned about how we would make the new keybinding discoverable.

**Comment by meganrogge** [comment]
I found this annoying as a mostly keyboard only user myself today when creating release notes.

**Comment by meganrogge** [comment]
Current flow:

- hear code action signal
- use code action keybinding `cmd+.`
- 🐛 paste widget opens
- hit escape
- use code action keybinding `cmd+.`
- 🐛 paste widget opens

https://github.com/user-attachments/assets/5fbdedd4-396b-4a82-8f0e-58634fbdd95c

**Comment by AFre100** [comment]
@meganrogge  So in terms of sound, would you say that it's an alert that communicates that a code action is available?

**Comment by meganrogge** [comment]
We have an alert/sound already to indicate there's a code action (the lightbulb). We need a new one for the paste widget. @mjbvz how does the snippet mode you mentioned yesterday play in here?

**Comment by meganrogge** [comment]
The sound is indicating the presence of the paste widget - currently, there's no indicaton of that to screen reader users.

This sound will be needed 😄 .

**Comment by meganrogge** [comment]
This is the paste widget FYI:

![Image](https://github.com/user-attachments/assets/fd2b4a88-b9f1-4482-b0b5-18d744520bea)

**Comment by AFre100** [comment]
Perfect -- really appreciate you specifying that!
